### PR TITLE
tools: update ticdc common maintenance

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? (Required)

Add maintenance document in TiCDC

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
@amyangfei Please fix the CI failure and involve a technical review.

**Comment by amyangfei** [comment]
> @amyangfei Please fix the CI failure and involve a technical review.

fixed, PTAL, thanks

**Comment by amyangfei** [comment]
PTAL @leoppro

**Comment by amyangfei** [comment]
PTAL @TomShawn thanks!

File: reference/tools/ticdc/deploy.md [hunk 1]
```diff
@@ -19,6 +19,15 @@ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_2.log --status-addr=127.
 cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --status-addr=127.0.0.1:8303
 ```
 
+对于 `cdc server` 命令中可用选项解释如下：
+
+- `gc-ttl`: TiCDC 在 PD 设置的服务级别 GC safepoint TTL 时长，单位为秒，默认值为 86400。
```
Comment (overvenus) [inline reference/tools/ticdc/deploy.md:24]:
```suggestion
- `gc-ttl`: TiCDC 在 PD 设置的服务级别 GC safepoint TTL 时长，单位为秒，默认值为 86400，即 24 小时。
```

File: reference/tools/ticdc/maintain.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+最新版本的 PD 支持外部服务设置服务级别的 GC safepoint，任何一个服务可以注册更新自己服务的 GC safepoint，PD 会保证任何小于该 GC safepoint 的 KV 数据不会在 TiKV 中被 GC。在 TiCDC 中启用了这一功能，用来保证 TiCDC 在不可用状况、或同步任务中断情况下，可以在 TiKV 内保留 TiCDC 需要消费的数据不被 GC。启动 CDC server 时可以通过 `gc-ttl` 指定 GC safepoint 的 TTL，这个值的含义是当 TiCDC 服务全部挂掉后，由 TiCDC 在 PD 所设置的 GC safepoint 保存的最大时间，该值默认为 86400 秒。
+
+如果同步任务长时间中断，累积未消费的数据比较多，初始启动 TiCDC 可能会发生 OOM。这种情况下可以启用 TiCDC 提供的文件排序功能，该功能会使用文件系统文件进行排序，（注意目前版本 TiCDC 还不支持动态修改文件排序和内存排序），启用的方式是创建同步任务时在 `cdc cli` 内传入 `--sort-engine=file` 和 `--sort-dir=/path/to/sort_dir`，使用示例如下
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235200 --sink-uri="mysql://root:123456@127.0.0.1:3306/" --sort-engine="file" --sort-dir="/data/cdc/sort"
+```
```
Comment (overvenus) [inline reference/tools/ticdc/maintain.md:51]: 麻烦补充下这种情况的原因和解决办法

```
[WARN] some tables are not eligible to replicate ...
```

**Pull request merged**